config/initializers/devise.rb

```diff
@@ -0,0 +1,263 @@
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` on Rails 4+ applications as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '3f5b2d773f5ab89255f0e3017a5684851d6c8b674c6d93342a9f6a619855dcc5e10f9601bcd6d76ee983a03f66747d603e8c142eff47f3bab037885f8fef2a4a'
```

Comment:
This key defaults to the Rails App secret_key_base and I am going to go with that.